examples/xdc-example/src/app/xdc.subscriptions.example.ts

```diff
@@ -0,0 +1,22 @@
+import { TatumXdcSDK } from '@tatumio/xdc'
+
+const xdcSDK = TatumXdcSDK({ apiKey: '75ea3138-d0a1-47df-932e-acb3ee807dab' })
+
+export async function xdcSubscriptionsExample() {
+  const id = await xdcSDK.subscriptions.createSubscription({
+    type: 'ACCOUNT_INCOMING_BLOCKCHAIN_TRANSACTION',
+    attr: {
+      id: '5e6be8e9e6aa436299950c41',
+      url: 'https://webhook.tatum.io/account',
+    },
+  })
+
+  await xdcSDK.subscriptions.deleteSubscription('5e68c66581f2ee32bc354087')
+  await xdcSDK.subscriptions.disableWebHookHmac()
+  await xdcSDK.subscriptions.enableWebHookHmac({
```

Comment:
add comment what is HMAC, link to the proper api calls in the API doc for all examples